frontend/.env.example

```diff
@@ -120,4 +120,10 @@ MOCK_AUTH_ALLOWED_REDIRECTS=http://localhost:3000/auth/callback/raoidc
 SCCH_BASE_URI=http://www.example.com
 
 # Community where the documents were generated from, and user belongs. This value is linked to the specific vault database for the given community.
-CCT_VAULT_COMMUNITY='SecurityReview'
\ No newline at end of file
+CCT_VAULT_COMMUNITY='SecurityReview'
+
+# Memoization: amount of time in milliseconds before the cache will expire (undefined for infinity)
+MEMO_MAXAGE_ALLPREFERREDLANGUAGES=0
+MEMO_MAXAGE_PREFERREDLANGUAGE=0
+MEMO_MAXAGE_ALLCOUNTRIES=0
+MEMO_MAXAGE_ALLREGIONS=0
```

Comment:
My suggestion here would be to add a time unit..
And it might be better to name the configs something a little more intuitive. (for example, `MEMO_MAXAGE` might not be as intuitive as `*_CACHE_TTL_SECONDS`)

```
LOOKUP_SVC_PREFERRED_LANGUAGES_CACHE_SECONDS
```

🤷

Reply:
Updated naming schema to use LOOKUP_SVC_******_CACHE_TTL_MILLISECONDS